Tiltfile

```
@@ -1,26 +1,54 @@
 # -*- mode: Python -*-
 
-# Read from the user's configuration file.
-settings = read_json('tilt-settings.json', default={})
+# set defaults
 
-allow_k8s_contexts(settings.get('allowed_contexts'))
-default_registry(settings.get('default_registry'))
+settings = {
+    "deploy_cert_manager": True,
+    "preload_images_for_kind": True,
+    "enable_providers": ["docker"],
+}
+
+# global settings
+settings.update(read_json(
+    "tilt-settings.json",
+    default = {},
+))
+
+allow_k8s_contexts(settings.get("allowed_contexts"))
 
+default_registry(settings.get("default_registry"))
 
 providers = {
-    'core': {
-        'context': '.',
-        'image': 'gcr.io/k8s-staging-cluster-api/cluster-api-controller',
-        'live_reload_deps': ['main.go', 'go.mod', 'go.sum', 'api', 'bootstrap', 'cmd', 'controllers', 'errors', 'util']
+    "core": {
+        "context": ".",
+        "image": "gcr.io/k8s-staging-cluster-api/cluster-api-controller",
+        "live_reload_deps": [
+            "main.go",
+            "go.mod",
+            "go.sum",
+            "api",
+            "bootstrap",
+            "cmd",
+            "controllers",
```

nit, and I can add in a followup, but this should now include `"controlplane",`
```suggestion
            "controllers",
            "controlplane",
```

---

Nice spot. Adding.